File: lua/test/exists_test.lua
-- BranchCustomExports SDK exists test

local sdk = require("branch-custom-exports_sdk")

describe("BranchCustomExportsSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { BranchCustomExportsSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await BranchCustomExportsSDK.test()
    equal(null !== testsdk, true)
  })

})
